include/aie/Dialect/AIE/IR/AIE.td

```diff
@@ -850,13 +850,14 @@ def AIE_DMABDOp: AIE_Op<"dmaBd", []> {
   // ];
 }
 
-def AIE_DMAStartOp: AIE_Op<"dmaStart", [ParentOneOf<["MemOp", "func::FuncOp", "ShimDMAOp"]>, Terminator]>,
+def AIE_DMAStartOp: AIE_Op<"dmaStart", [ParentOneOf<["MemOp", "MemTileDMAOp", "func::FuncOp", "ShimDMAOp"]>, Terminator]>,
                      Results<(outs I1:$valid)> {
 
   let summary = "An op to start DMA";
   let description = [{
     This operation declares a DMA channel to be used for data transfer. It usually exists inside
-    either a MemOp (representing a TileDMA), or in a ShimDMAOp (representing a ShimDMA).
+    either a MemOp (representing a TileDMA), a MemTileDMAOp (representing a DMA in a memtile),
```

Comment:
I do think at some point we should rename the MemOp to something more like ComputeTileDMAOp or maybe just DMAOp.

Reply:
I agree, but let's deal with that separately.